flashcards/backend/decks/models.py

```diff
@@ -1,6 +1,39 @@
 from django.db import models
 
 
+class Flashcard(models.Model):
+    pass
+
+
+class Deck(models.Model):
+    pass
+
+
+class Tag(models.Model):
+    """Model for representing tags for decks
+
+    :param name: Tag name
+    :type name: str
+    :param description: Description for tag
+    :type description: str
+    :param flashcard: Foreign Key to model Flashcard
+    :type flashcard: int
+    :param deck: Foreign Key to model Deck
+    :type deck: int
+
+    """
+
+    name = models.CharField(max_length=24)
```

Comment:
imo, we don't need 200 instances of the same tag 'python' stored in db. Maybe we should set unique=True and override the save method to return existing row if it already exists in the db? 🤔

Reply:
I agree, there could be also an annotation that user input should be processed to lowercase

Reply:
Ok, I tested, unique is enough :)

Reply:
meaning that it checks uniqueness for both `python` and `Python`?